spec/models/calculator_spec.js

```diff
@@ -382,7 +385,7 @@ describe('Calculator', () => {
     });
   });
 
-  it('multiple observation measure correctly', () => {
+  xit('multiple observation measure correctly', () => {
```

Comment:
Should this skipped test have a TODO comment like the others?

Reply:
Dont think it's necessary--It wouldn't hurt to have the comment, but adding commits on top of this PR for just a comment doesn't make sense IMO, as the code was reviewed in the PR into the feature branch